src/main/java/com/palantir/gradle/revapi/RevapiPlugin.java

```diff
@@ -60,14 +61,40 @@ public void apply(Project project) {
                     Configuration revapiNewApi = project.getConfigurations().create("revapiNewApi", conf -> {
                         conf.extendsFrom(
                                 project.getConfigurations().getByName(JavaPlugin.COMPILE_CLASSPATH_CONFIGURATION_NAME));
+                        conf.setCanBeConsumed(false);
```

Comment:
actually why do we craete a new configuration here, rather than just use `compileClasspath`?

Reply:
I think it's because we originally resolved runtime elements until a few weeks ago - probably don't need to at all now :)